### DISCUSSION TITLE
Weird Error

### DISCUSSION THREAD

**Discussion by IRediTOTO** [vercel/next.js#14817] [Help] [answered]
# Bug report

## Describe the bug
Hi, i am working on my app, but it get an error without no console error. 
![image](https://user-images.githubusercontent.com/15695252/86477402-8eec6580-bd72-11ea-90e4-662117a04277.png)

App just **Freeze**.

- I think it come from my PC, but I upload code to server, build complete but freeze too.

- I tried to delete all npm and next foleder then npm install again, still freeze!

- I think problem from my code, but I still worked with it yesterday, newest package I install is Axios. I don't think Axios can make app down, atleast still give some error on console, right? No api, no page run. I really have no idea for this. :(

 _What my mistake can make app freeze?_  Thank you for read this.

```
ready - started server on http://localhost:3001
info  - Loaded env from D:\Job\My-app\.env
Warning: Built-in CSS support is being disabled due to custom CSS configuration being detected.
See here for more info: https://err.sh/next.js/built-in-css-disabled

event - compiled successfully
event - build page: /
wait  - compiling...
event - compiled successfully
```

## System information

- OS: Windows
- Version of Next.js: 9.4.4
- Version of Node.js: 12x

**Comment by M1ck0** [comment]
Could you provide GitHub repo with the code of your app?

**Comment by IRediTOTO** [comment]
@M1ck0  Thank you for replie me, my project codes in many months, I don't want share it, hope u understand. 
I don't see any reason why app down. Hope someone get this too.

**Comment by M1ck0** [comment]
Okay. I do understand.

What would I suggest you do. Exclude the main parts of the code. Comment out biggest components in your app and then try to recompile it. Remove ```node_modules``` and ```yarn.lock``` or ```package-lock.json``` from your project and then install packages again.

Try to exclude as much as possible to see what is going on. Then come here if it doesn't work.

**Comment by IRediTOTO** [comment]
Just tried, freeze. 
Here is all my package 
```
"dependencies": {
      "@paypal/checkout-server-sdk": "^1.0.2",
      "@tinymce/tinymce-react": "^3.6.0",
      "@zeit/next-css": "^1.0.1",
      "@zeit/next-sass": "^1.0.1",
      "@zeit/next-source-maps": "0.0.3",
      "algoliasearch": "^4.3.0",
      "animate.css": "^3.7.2",
      "archiver": "^4.0.1",
      "archiver-zip-encrypted": "^1.0.8",
      "axios": "^0.19.2",
      "bulma": "^0.8.2",
      "bulma-helpers": "^0.3.12",
      "ckeditor5": "^19.1.1",
      "connect-mongo": "^3.2.0",
      "cors": "^2.8.5",
      "dotenv": "^8.2.0",
      "form-data": "^3.0.0",
      "formidable": "^1.2.2",
      "googleapis": "^52.1.0",
      "jsonwebtoken": "^8.5.1",
      "moment": "^2.27.0",
      "mongodb": "^3.5.9",
      "nanoid": "^3.1.10",
      "next": "^9.4.4",
      "next-connect": "^0.6.6",
      "next-images": "^1.4.0",
      "next-purgecss": "^4.0.0",
      "next-seo": "^4.7.1",
      "next-session": "^3.2.3",
      "node-fetch": "^2.6.0",
      "node-pre-gyp": "^0.12.0",
      "node-sass": "^4.14.1",
      "nprogress": "^0.2.0",
      "passport": "^0.4.1",
      "passport-facebook": "^3.0.0",
      "passport-google-oauth": "^2.0.0",
      "passport-jwt": "^4.0.0",
      "passport-local": "^1.0.0",
      "passport-twitter": "^1.0.4",
      "publitio_js_sdk": "^1.1.1",
      "react": "^16.13.1",
      "react-dom": "^16.13.1",
      "react-dropzone": "^11.0.1",
      "react-facebook": "^8.1.4",
      "react-google-recaptcha": "^2.1.0",
      "react-google-recaptcha-v3": "^1.5.2",
      "react-hook-form": "^5.7.2",
      "react-instantsearch-dom": "^6.6.0",
      "react-notifications-component": "^2.4.0",
      "react-table": "^7.2.1",
      "sib-api-v3-sdk": "^7.2.3",
      "swr": "^0.2.3",
      "tailwindcss": "^1.4.6"
   },
   "devDependencies": {},
   "cacheDirectories": [
      "node_modules/",
      ".next/cache/"
   ]
```
On server, it freeze too. 
![image](https://user-images.githubusercontent.com/15695252/86487976-c9153180-bd89-11ea-8f86-8189141381c3.png)
I don't know what code I did wrong to find and fix
In console, just freeze like this, no more, no less, event I enter localhost:3001 on browser
![image](https://user-images.githubusercontent.com/15695252/86488206-6cfedd00-bd8a-11ea-83dc-f9af77678a07.png)

My app don't have anything special, just nextjs,api, query mongo,..  Here is index.js
```
export default (props) => {
   return (
      <Layout>
         <div className="hero bg-purple">
            <section className="hero is-medium is-primary is-bold">
               <div className="hero-body">
                  <div className="container has-text-centered is-center">
                     <h2 className="title">
                       Find any products
                     </h2>
                     <div className="columns">
                        <div className="column is-three-fifths is-offset-one-fifth ">
                           <AlgoliaSearch />
                        </div>
                     </div>
                  </div>
               </div>
            </section>
         </div>
         {props.items && (
            <ItemsContainer
               initialData={props.items}
               fetcherUrl={"/api/items"}
               urlQuery={`limit=8`}
               limit={8}
               loadMore={true}
            />
         )}
      </Layout>
   );
};

export const getStaticProps = async () => {
   try {
      const items = await getItems(await mongoDb(), 0, 8);
      if (items && !items.error) {
         return {
            props: {
               items,
            },
            //   unstable_revalidate: 1,
         };
      } else {
         return {
            props: { items: [] },
            //   unstable_revalidate: 1,
         };
      }
   } catch (error) {
      if (error) {
         console.error(
            "c7f930fa-eac3-4145-9dcf-921cb202d688 error ",
            error
         );
         return {
            props: { items: [] },
            //      unstable_revalidate: 1,
         };
      }
   }
};
```

**Comment by BranonConor** [comment]
Hey guys, I'm doing a code along MERN Stack project using NextJS on the front end and I'm getting the same problem. I don't think it's my backend, I've replaced all my code with the instructor's to be sure - and one second I was successfully writing data to the backend and then all of a sudden I couldn't even visit a page as the whole front end would freeze up with an endless request for a certain page, just like OP's problem. 

Happy to share my code if it would be helpful. Hope we can figure this one out!

**Comment by IRediTOTO** [comment, marked as answer]
@BranonConor  This error just apear today. Do you got it today too?

**Reply by BranonConor** [reply]
@warcraft14115 I actually got it yesterday around 4:00 pm Pacific Time. Clocked in to fix it today and can't figure it out after another hour or so. I'm almost certain it's a NextJS problem as I can write data to my database still from Postman. I just can't render the page in my UI that offers that functionality - it gets hung up forever like yours. Weird!

**Reply by IRediTOTO** [reply]
I think problem come from npm package. :( Lost many hours to fix. Just tried other nextjs, still working fine. 
I just don't understand what can make app freeze like this, can't call api , can't render page ...

**Reply by IRediTOTO** [reply]
I think I found it, something stuck in mongdb connect. :| This is not from nextjs :\ it from mongo server. Let's try on your side

**Reply by BranonConor** [reply]
Interesting - how exactly did you fix it? I'll try your fix on my end and see if it works!

**Reply by IRediTOTO** [reply]
:( Still freeze. I must crazy for this.
![image](https://user-images.githubusercontent.com/15695252/86495516-e9ea8080-bda3-11ea-91f8-dee7b40107f8.png)

**Reply by IRediTOTO** [reply]
@BranonConor I found it, error from a regex func. I alway got error with this. You should check your logic code.
